src/itest/kotlin/nl/info/zac/itest/ZaakafhandelParametersRestServiceTest.kt

```diff
@@ -100,15 +100,15 @@ class ZaakafhandelParametersRestServiceTest : BehaviorSpec({
                 responseBody shouldEqualJsonIgnoringOrder """
                     [ 
                         {
-                          "id" : -1,
+                          "id" : "-1",
```

Comment:
These values seem to drop out of thin air, is this from a database initalisation script? Or where does this come from?

Reply:
Yes, these are part of provisioning scripts.

Reply:
Added integration tests constants for these